bodhi/server/push.py

```diff
@@ -71,15 +71,15 @@ def push(username, cert_prefix, **kwargs):
     db_factory = transactional_session_maker()
     composes = []
     with db_factory() as session:
-        if not resume and session.query(Compose).count():
-            click.confirm('Existing composes detected: {}. Do you wish to resume them all?'.format(
-                ', '.join([str(c) for c in session.query(Compose).all()])), abort=True)
+        if not resume and session.query(Compose).filter(Compose.state == ComposeState.failed).count():
+            click.confirm('Failed composes detected: {}. Do you wish to resume them all?'.format(
+                ', '.join([str(c) for c in session.query(Compose).filter(Compose.state == ComposeState.failed).all()])), abort=True)
```

Comment:
I would probably make the query match non-failed composes instead of detecting failed ones. We could then print a warning and exit, something like "WARNING: running composes detected, refusing to resume as that is a dangerous operation. If you are sure they are not really running, you can re-run with the --force flag." Of course, this would require a ```--force``` flag to be added as well.